packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -714,6 +724,16 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return filterValueReset;
 	}
 
+	/**
+	 * Gets the class for a specific row if a rowClassNameFn function has been given as an Input
+	 * @param row - The data object passed to the row.
+	 * @param index - The index of the row.
+	 * @returns The className generated by the rowClassNameFn function
+	 */
+	public getRowClass(row: any, index: number): string {
+		return this.rowClassNameFn && typeof this.rowClassNameFn === "function" ? this.rowClassNameFn(row, index) : "";
```

Comment:
I think the first check can be removed... `this.rowClassNameFn && typeof this.rowClassNameFn === "function"``could be simply `typeof this.rowClassNameFn === "function"``;)

Reply:
I thought that we could only do `if (this.rowClassNameFn instanceof Function)` if we want to check if it is a function...
Which way should we use so ? :blush:
I've the feeling that `typeof this.rowClassNameFn === "function"` is better.

Reply:
true, that is what I am also suggesting... but the formatting of my comment was wrong :( I changed it

Reply:
True, but I suggest to do it in a different PR where we can refactor all the code not only for this but for other things like `readonly`, `private`, `get`, `set`, etc...